src/nix/optimise-store.cc

```diff
@@ -27,3 +27,4 @@ struct CmdOptimiseStore : StoreCommand
 };
 
 static auto rCmdOptimiseStore = registerCommand2<CmdOptimiseStore>({"store", "optimise"});
+static auto rCmdOptimizeStore = registerCommand2<CmdOptimiseStore>({"store", "optimize"});
```

Comment:
(Not commenting on whether we want American-spelling aliases for commands.)

The correct way to create an alias is to add it `aliases` in `src/nix/main.cc`. That way it doesn't duplicate manpages etc., and there is one "canonical" spelling.

Reply:
Got it. Thanks for pointing me in the right direction.

On first glance an alias like this would work (`nix optimize -> nix store optimise`)

```
{"optimize", { AliasStatus::AcceptedShorthand, {"store", "optimise"}}},
```

but neither of these do intending `nix store optimise -> nix store optimize`: 

```
{"store optimize", { AliasStatus::AcceptedShorthand, {"store", "optimise"}}},
{"optimize", { AliasStatus::AcceptedShorthand, {"optimise"}}},
```

Seems to be because `rewriteArgs` expects a one-word command (unless you quote the argument) and doesn't modify the sub-command. I'll see if I can figure something else out.